lib/loader.mjs

```diff
@@ -0,0 +1,20 @@
+import { createRequire } from 'module';
+import { fileURLToPath } from 'url';
+import { send } from './ipc.mjs';
+
+const require = createRequire(import.meta.url);
+
+export async function load(url, context, defaultLoad) {
+  const getPackageType = require('get-package-type');
```

Comment:
It looks like this is only used in the unknown file extension case. Let's get it out of the critical path.